Glyssen/Rules/CharacterGroupGenerator.cs

```diff
@@ -914,13 +1021,118 @@ public static void DistributeBooksAmongNarratorGroups(List<AuthorStats> authorSt
 				}
 			}
 
-			private void AssignExtraBiblicalCharactersToExtraBiblicalGroups(List<CharacterDetail> includedCharacterDetails)
+			private void AssignExtraBiblicalCharactersToExtraBiblicalGroups(List<CharacterDetail> includedCharacterDetails,
+				ProjectDramatizationPreferences dramatizationPreferences)
+			{
+				var narratorPrefix = CharacterVerseData.GetCharacterPrefix(CharacterVerseData.StandardCharacter.Narrator);
```

Comment:
Rather than passing narrator prefix down through all these layers of private methods (including possible paths where it might never be needed), why not just get it once at the one line of code that uses it?